packages/admin/admin/src/error/inlineError/InlineError.tsx

```diff
@@ -0,0 +1,128 @@
+import { Error, Info, Warning } from "@comet/admin-icons";
+import { type ComponentsOverrides, type Theme, type Typography } from "@mui/material";
+import { useThemeProps } from "@mui/material/styles";
+import { type FunctionComponent, type PropsWithChildren, type ReactNode } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+import { ActionsContainer, Description, IconContainer, Root, Title } from "./InlineError.styles";
+
+export type InlineErrorClassKey = "root" | "iconContainer" | "title" | "description" | "actionsContainer";
+
+type InlineErrorVariant = "error" | "warning" | "info";
+
+type InlineErrorProps = ThemedComponentBaseProps<{
+    root: "div";
```

Comment:
Since this slot extends `Box` please use `typeof Box` or extend from `"div"` in the styled component. 
This is so the type of the supported props and actually supported props match.

Reply:
i missed that. Changed it.